portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@ $("#comment-form").submit(function(event){
 	event.preventDefault();
 	var postUrl = "/comments";
 	var formData = $(this).serialize();
-
+  this.reset();
```

Comment:
May want to fix the indentation

Reply:
That's weird, it looked fine on my screen, thanks for pointing it out